src/main/java/org/zephyrsoft/sdb2/model/SongsModelController.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of the Song Database (SDB).
+ *
+ * SDB is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * SDB is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with SDB. If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
Please not that SDB ist licensed under GPL3 without option of using a later version. This was changed two months ago. The file header should be this:

```
/*
 * This file is part of the Song Database (SDB).
 *
 * SDB is free software: you can redistribute it and/or modify it
 * under the terms of the GNU General Public License 3.0 as published by
 * the Free Software Foundation.
 *
 * SDB is distributed in the hope that it will be useful, but
 * WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 * GNU General Public License 3.0 for more details.
 *
 * You should have received a copy of the GNU General Public License 3.0
 * along with SDB. If not, see <http://www.gnu.org/licenses/>.
 */
```

Reply:
Fixed it. We have a gpl2 under src/dist because of gpl2 dependencies right?

Reply:
No. That's because I forgot to change it there too when I changed in it the project's root directory. It's fixed now. Thanks for the hint! :wink: